.editorconfig

```diff
@@ -7,9 +7,8 @@ indent_style = space
 indent_size = 2
 insert_final_newline = true
 trim_trailing_whitespace = true
-
-[*.ts]
-quote_type = single
+end_of_line = lf
+max_line_length = 120
```

Comment:
isn't 120 too long??? I think Prettier recommended 90

Reply:
changed to 90